File: 프로그래머스/lv1/86491. 최소직사각형/최소직사각형.java
class Solution {
    public int solution(int[][] sizes) {

        int max_width = 0;
        int max_height = 0;
        
        for(int i = 0; i<sizes.length; i++){
            int w = Math.max(sizes[i][0], sizes[i][1]);
            int h = Math.min(sizes[i][0], sizes[i][1]);
            
            max_width = Math.max(max_width, w);
            max_height = Math.max(max_height, h);
        }
        
        
        return max_width * max_height;
    }
}

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    
    let max_width = 0, max_height = 0;
    
    for(let i= 0; i<sizes.length; i++){
        let width = Math.max(sizes[i][0], sizes[i][1]);
        let height = Math.min(sizes[i][0], sizes[i][1]);
        
        max_width = Math.max(width, max_width);
        max_height = Math.max(height,max_height);
    }
    
    return max_height * max_width;
}